lib/mix/lib/releases/assembler.ex

```diff
@@ -497,6 +505,12 @@ defmodule Mix.Releases.Assembler do
       do: :ok
   end
 
+  defp generate_cookie do
+    # 255 is max. atom length, see http://erlang.org/doc/efficiency_guide/advanced.html
+    # Cookies, at least when read from disk, need to be from the printable ASCII range
+    (1..255)|> Enum.map(fn _ -> Enum.random(?\s..?~) end) |> List.to_atom
```

Comment:
From https://github.com/erlang/otp/blob/3b7a6ffddc819bf305353a593904cea9e932e7dc/lib/kernel/src/auth.erl#L351 it looks like space chars should not be a problem, but I'm totally okay with sacrificing one character, especially since it might be a smoother experience with tooling like editors and such.

With 64 chars from the suggested range, we are at around 418 bits, compared to ~94 bits generated by https://github.com/erlang/otp/blob/3b7a6ffddc819bf305353a593904cea9e932e7dc/lib/kernel/src/auth.erl#L376. However, I would consider using http://erlang.org/doc/man/crypto.html#strong_rand_bytes-1, since the seeding from :rand looks like it has very little entropy (it basically just uses time, which is considered to be feasible for brute-forcing)